src/components/dashboard/EditProject.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+    Upload,
+    getValueFromEvent,
+    useApiUrl,
+    Edit,
+    Form,
+    Input,
+    useForm,
+    ImageField,
+} from "@pankod/refine";
+
+const formInputs = [
+    { label: "Slug", name: "slug", required: true },
+    { label: "CatchPhrase", name: "catchPhrase", required: true },
+    { label: "Title", name: "title", required: true },
```

Comment:
The second key of these three array objects is "name" but the rest is "title". we have to be consistent with those names so either pick "title" or "name"